apps/happy-blocks/src/pricing-plans/config.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Load and export the pricing plans block configuration provided from the backend.
+ */
+const config = {
+	plans: [ 'business', 'business-monthly', 'ecommerce', 'ecommerce-monthly' ],
```

Comment:
I think there are constants for this within Calypso but not sure whether we should or even could reuse them.

Reply:
I think it's in `client/lib/plans/features-list.js` but you still need to define the list of plans you want to show, just that you could use the constants from there instead of strings — if possible to reuse, of course. If not, disregard.